tests/unittest_inference.py

```diff
@@ -6498,7 +6497,9 @@ def __init__(self, *args, **kwargs):
     assert inferred is util.Uninferable
 
 
-def test_inferring_properties_multiple_time_does_not_mutate_locals_multiple_times() -> None:
+def test_inferring_properties_multiple_time_does_not_mutate_locals_multiple_times() -> (
+    None
```

Comment:
This feels very weird?

Reply:
Black is trying to make the line length smaller by any mean necessary :)